src/db/prisma/schema.prisma

```diff
@@ -0,0 +1,72 @@
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "postgresql"
+  url      = env("DATABASE_URL")
+}
+
+model Job {
+  id              String          @id @default(dbgenerated("gen_random_uuid()")) @db.Uuid
```

Comment:
Consider using the `uuid()` instead of `dbgenerated` (needs the implemetation as `updatedAt`)